test/unit/math/opencl/prim/mdivide_left_tri_low_test.cpp

```diff
@@ -0,0 +1,121 @@
+#ifdef STAN_OPENCL
+#include <stan/math/prim/mat.hpp>
+#include <stan/math/opencl/opencl.hpp>
+#include <test/unit/math/expect_near_rel.hpp>
+#include <boost/random/mersenne_twister.hpp>
+#include <gtest/gtest.h>
+#include <algorithm>
+
+#define EXPECT_MATRIX_NEAR(A, B, DELTA) \
```

Comment:
These tests and the mdivide_right_tri_low follow the same idea as the OpenCL tests we have for math/prim/mdivide_left_tri